packages/tooling/task-runner/rfc/design-fspy-windows-detours.md

```
@@ -0,0 +1,137 @@
+```ts
+interface FileAccess {
+    path: string;
+    type: "missing" | "read" | "readdir" | "stat" | "write";
+}
+```
+
+The DLL emits these events; the supervisor reads them and folds them into `TrackingResult.accesses` exactly like the strace branch does.
+
+### Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)